sound/soc/sof/virtio-miscdev.c

```diff
@@ -0,0 +1,255 @@
+// SPDX-License-Identifier: (GPL-2.0 OR BSD-3-Clause)
+/*
+ * This file is provided under a dual BSD/GPLv2 license.  When using or
+ * redistributing this file, you may do so under either license.
+ *
+ * Copyright(c) 2017 Intel Corporation. All rights reserved.
+ *
+ * Author: Libin Yang <libin.yang@intel.com>
+ *         Luo Xionghu <xionghu.luo@intel.com>
+ *         Liam Girdwood <liam.r.girdwood@linux.intel.com>
+ */
+
+#include <linux/module.h>
+#include <linux/miscdevice.h>
+#include <linux/fs.h>
+#include <linux/file.h>
+#include <linux/slab.h>
+
+#include "sof-priv.h"
+#include "virtio-be.h"
+#include "virtio-miscdev.h"
+#include <linux/vbs/vbs.h>
+
+/*
+ * This module registers a device node /dev/vbs_k_audio,
+ * that handle the communication between Device Model and
+ * the virtio backend service. The device model can
+ * control the backend to : set the status, set the vq account and etc.
+ * The config of the DM and VBS must be accordance.
```

Comment:
fix line length - which userspace API doe this use - Pls add link.

Reply:
I don't understand this comment. What shall I do here? Make the line shorter?